### ISSUE TITLE
What is the Tensorflow version used in this repo? Is it based on TF2?

### ISSUE THREAD

**Issue by Abdul-Mukit** [amanchadha/coursera-deep-learning-specialization#12]
I am getting the following error message in the TensorFlow_Tutorial_v3b.ipynb

AttributeError: module 'tensorflow' has no attribute 'global_variables_initializer'

I am guessing this repo uses an older version of the TF?

**Comment by abdur75648** [comment]
Yes it uses the TF1 because the specialisation initially used TF1. It was in 2021 that they shifted to TF2 and made some other necessary changes too.
Please have a look at this repo: [Deep Learning Specialization [Updated Version 2021]](https://github.com/abdur75648/Deep-Learning-Specialization-Coursera)
It contains the latest assignments from updated labs. I hope this help :)
